### PR TITLE
introduce notification toaster

### LINKED ISSUE #44: Handle config.topics.error with error message

### PR DESCRIPTION
**Pull request by steveoh**
publish to topics.toast.
if you send a string it defaults to info.
if you send {message: type:} you can use info, success, warning, danger
closes #44

### CODE REVIEW

**Comment by steveoh** [comment]
this might end up in ijit or something, what do you think?

File: src/app/Toaster.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        //      a toaster notification widget
+        templateString: template,
+        baseClass: 'toaster',
+
+        // separator: Number
+        //		The number if items currently being shown
+        count: 0,
+
+        // separator: Number
+        //		The amx number if items to show
```
Comment (stdavis) [inline src/app/Toaster.js]: typo's

File: src/app/Toaster.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+	lang
+) {
+    return declare([_WidgetBase, _TemplatedMixin], {
+        // description:
+        //      a toaster notification widget
+        templateString: template,
+        baseClass: 'toaster',
+
+        // separator: Number
+        //		The number if items currently being shown
```
Comment (stdavis) [inline src/app/Toaster.js]: typo

File: src/app/Toaster.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            //      handles the message and sets default arguments
+            // {message: type:}
+            console.log('app.Toaster:handleMessage', arguments);
+
+            if (lang.isString(message)) {
+                this.setContent(message, this.defaultClass);
+            } else {
+                var cssClass = message.type;
+                if (!cssClass || this.allowableClasses.indexOf(cssClass) < 0) {
+                    cssClass = this.defaultClass;
```
Comment (stdavis) [inline src/app/Toaster.js]: I wonder if this should throw a console warning or error.

**Comment by stdavis** [comment]
Closed with ecada41.

**Pull request closed without merging**